src/ecpool_worker.erl

```diff
@@ -131,8 +133,11 @@ init([Pool, Id, Mod, Opts]) ->
     case connect_internal(State) of
         {ok, NewState} ->
             gproc_pool:connect_worker(ecpool:name(Pool), {Pool, Id}),
+            send_initial_connect_response(InitialConnectResultReceiver, ok),
             {ok, NewState};
-        {error, Error} -> {stop, Error}
+        Error -> 
+            send_initial_connect_response(InitialConnectResultReceiver, Error),
+            {ok, State}
```

Comment:
there is no reason to keep this worker running?
return `ignore` here should do.

Reply:
Ok, will test if this results in a crash report or not.

Reply:
ignore does not result in a crash report so I have changed this to return ignore according to your suggestion: https://github.com/emqx/ecpool/pull/50/commits/bf64597eb9cc4765c2be0f40b56f97a69d2e2ded